draft-ietf-tls-tls13.md

```diff
@@ -3993,11 +3993,11 @@ In the absence of an application profile standard specifying otherwise, a
 TLS-compliant application MUST implement the following TLS extensions:
 
   * Supported Versions ("supported_versions"; {{supported-versions}})
+  * Cookie ("cookie"; {{cookie}})
```

Comment:
https://tools.ietf.org/html/draft-ietf-tls-tls13-17#section-8.2

When linkified, it currently goes Section 4.2.1, 4.2.3, 4.2.4, 4.2.5, 4.2.6, 4.2.2. Just putting them in order. (as I said, just a nitpick)